README.md

```diff
@@ -19,66 +25,56 @@ All the files refers to the `SLUS-00067` version of the game.
-
-## Build
-
-1. You need `gcc-mipsel-linux-gnu` that you can easily install on any Debian-based Linux distribution. On Windows it is highly recommended to just use Ubuntu with WSL
-1. Copy the game's data from your SOTN game copy into a new folder in the root directory of the repository called `iso`
-1. Run `make extract` to generate the assembly files
-1. Run `make all` to compile the binaries into the `build/` directory
-
-## Restore MAD (debug room)
-
-The debug room overlay `ST/MAD.BIN` was compiled earlier than the first retail release of the game. All the offsets that refers to `DRA.BIN` points to invalid portions of data or to the wrong API calls, effectively breaking the majority of its original functionalities. That is why the debug room does not contain any object. By compiling the debug room with `make mad_fix` you can restore it by redirecting the old pointers to the retail version of the game. 
-
-Be aware that not all the offsets have been yet redirected, so it will still be not entirely functional until further update.
-
-## Non-matching build
 
-Some non-matching functions are present in the source code by disabled by the macro `NON_MATCHING`. You can still compile the game binaries by running ` CPP_FLAGS=-DNON_MATCHING make`. In theory they might be logically equivalent in-game, but I cannot promise that. Few of them could match by tuning or changing the compiler.
-
-## Check function matching
-
-With [asm-differ](https://github.com/simonlindholm/asm-differ) you can check if a funtion written in C matches its assembly counterpart.
-
-1. Be sure to resolve the submodule with `git submodule update --init`
-1. Ensure to create a matching binary with `make clean && make extract && make all && mkdir expected && cp -r build expected/`
-1. Choose a function to match (eg. `func_8018E964`), an overlay (eg. `st/mad`) and then invoke `python3 ./tools/asm-differ/diff.py -mwo --overlay st/mad func_8018E964`
-
-## Decompile and contribute
-
-This guides you step-by-step to contribute to the decompilation project.
-
-1. Choose an overlay to work with (eg. `ST/WRP`)
-1. Look for one of those function which hasn't successfully decompiled yet (eg. `INCLUDE_ASM("asm/st/wrp/nonmatchings/6FD0", func_801873A0);`)
-1. Follow [the guide to check the function matching](#check-function-matching)
-1. Look for its assembly file (eg. `asm/st/wrp/nonmatchings/6FD0/func_801873A0.s`)
-1. Run `ASSEMBLY=asm/st/wrp/nonmatchings/6FD0/func_801873A0.s make decompile` to dump the decompiled code into `ctx.c.m2c`
-1. Replace the `INCLUDE_ASM` you targeted, replace it with the content of `ctx.c.m2c` and [tweak the code with `asm-differ`](#check-function-matching)
-1. You will probably have some differences from your compiled code to the original; keep refactoring the code and move variables around until you have a 100% match.
+## How to
+* setup the project (assuming linux ubuntu/debian or windows with WSL)
```

Comment:
I think we can either do
```
## How to

### Setup the project

blabla

### Build

and so on
```

or
```
## How to setup the project

blabla

## How to build

and so on
```

rather than having a list of lists. I believe this way will be a bit easier to read and digest.